galsim/bandpass.py

```diff
@@ -278,15 +292,12 @@ def __div__(self, other):
         else:
             tp = lambda w: self.func(w) / other
 
-        return Bandpass(tp, wave_type, blue_limit, red_limit, _wave_list=wave_list)
+        # Note that .zeropoint attribute gets reset by __div__
```

Comment:
Yep.  No propagation of zeropoints for either multiplication or division.  I've updated both the `Bandpass` and `Bandpass.withZeropoint` docstrings as suggested.